integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -1392,6 +1394,8 @@ protected void initialize(Map<String, Object> inputDomainMap) throws Exception {
     TestUtils.exec(
         "cp -rf " + BaseTest.getProjectRoot() + "/kubernetes/samples " + BaseTest.getResultDir());
 
+    copyDomainTemplate(inputDomainMap);
```

Comment:
pls move this call to callCreateDomainScript() instead of initialize()

Reply:
done